src/Components/Blueprints/BlueprintsSideBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, Dispatch, SetStateAction } from 'react';
+
+import {
+  EmptyState,
+  EmptyStateActions,
+  EmptyStateBody,
+  EmptyStateFooter,
+  EmptyStateHeader,
+  EmptyStateIcon,
+  SearchInput,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { Link } from 'react-router-dom';
+
+import BlueprintCard from './BlueprintCard';
+
+import { BlueprintItem } from '../../store/imageBuilderApi';
+import { resolveRelPath } from '../../Utilities/path';
+
+interface blueprintProps {
```

Comment:
Convention is to use a `Type` and not `interface`.

Reply:
This should also match the component name, so `type BlueprintsSidebarProps = { ... }`.